testcases/multizone_office_simple_hydronic/doc/index.html

```diff
@@ -0,0 +1,1206 @@
+The internal controls of the AHUs (not connected to the BMS) manage the
+internal 3-way valves in the heating/cooling coils and the recovery by-pass to achieve the desired air supply set-point.
+The by-pass control of the AHUs is connected to the BMS,
+unlocking the possibility of free cooling during nights.
+However, this feature is not implemented in the baseline controller.
+</p>
+
+<p>
+The mixing 3-way valves in the distribution system modulate to track
+a given supply set-point.
+For the ventiloconvectors, the set-points follow a heating/cooling curve
+based only in the outdoor temperature (no room compensation is included).
+<ul>
+<li>The heating curve of the ventiloconvector heating coil takes the following points:</li>
+	<ul>
+	<li>70 &#176;C when the outdoor temperature is -10 &#176;C</li>
+	<li>40 &#176;C when the outdoor temperature is 20 &#176;C</li>
+	</ul>
+<li>The cooling curve of the ventiloconvector cooling coil takes the following points:</li>
+	<ul>
+	<li>7 &#176;C when the outdoor temperature is 35 &#176;C</li>
+	<li>12 &#176;C when the outdoor temperature is 15 &#176;C</li>
+	</ul>
+</ul>
+
+
+For the ventilation coils, these set-points are set constant to:
+<ul>
+<li>50 &#176;C for the heating coil in the AHUs</li>
+<li>9 &#176;C for the heating coil in the AHUs</li>
+</ul>
+The modulation is achieved by means of PI controllers.
+All PIs have a constant gain of 0.2 and an integral gain of 30s.
+The model allows to override either the set-point to be tracked by the PI
+or the valve position directly.
+</p>
+
+<p>
+The production system ideally tracks a given supply temperature set-point.
+The heating production system follows the maximum of the heating distribution set-points.
+The cooling production system follows the minimum of the cooling distribution set-points.
+The pumps in the production system are activated whenever at least one of the
+distribution pumps is activated.
+</p>
+
+
+<h3>Model IO's</h3>
+<h4>Inputs</h4>
+The model inputs are:
+<ul>
+<li>
+<code>bms_oveByPassNz_u</code> [1] [min=0.0, max=1.0]: North zone AHU setpoint to override the recovery bypass (for night free cooling purposes)
+</li>
+<li>
+<code>bms_oveByPassSz_u</code> [1] [min=0.0, max=1.0]: South zone AHU setpoint to override the recovery bypass (for night free cooling purposes)
+</li>
+<li>
+<code>bms_ovePrfAhuCooNz_u</code> [1] [min=0.0, max=1.0]: North zone AHU cooling circuit activation setpoint
+</li>
+<li>
+<code>bms_ovePrfAhuCooSz_u</code> [1] [min=0.0, max=1.0]: South zone AHU cooling circuit activation setpoint
+</li>
+<li>
+<code>bms_ovePrfAhuHeaNz_u</code> [1] [min=0.0, max=1.0]: North zone AHU heating circuit activation setpoint
+</li>
+<li>
+<code>bms_ovePrfAhuHeaSz_u</code> [1] [min=0.0, max=1.0]: South zone AHU heating circuit activation setpoint
+</li>
+<li>
+<code>bms_ovePrfAhuRetNz_u</code> [1] [min=0.0, max=1.0]: North zone AHU return fan setpoint
```

Comment:
Suggest adding "fan **speed** set point" for all

Reply:
Suggestion accepted

Reply:
Got it, thanks!